test/acceptance/ResourcesTest.js

```diff
@@ -0,0 +1,50 @@
+Feature("Resources Page");
+
+Scenario("Hides Results Which Do Not Match Search Query", ({ I }) => {
+	I.amOnPage("https://codesupport.dev/resources");
```

Comment:
Are these not going to run on live?

What happens if we open a PR to change something, how would the tests reflect this?

Reply:
Yeah, valid point. I guess I was thinking they would let you know if the site broke, but it's a bit late then... Makes sense for them to run on the branch only?

Reply:
Yeah on the branch. Unsure how that will work in the CI/CD pipeline as we'd essentially need to spin up the current version (branch) and test - along with the backend, which doesn't want to be the live API

Reply:
I'm guessing we just use the output of the `build` step and run it against that, seems like we can mock the API via Codecept (see [docs](https://codecept.io/puppeteer/#mocking-requests)). I think doing that is fine given we’re testing frontend functionality not backend responses?

Reply:
Sounds like a good plan 👍